ch8/quantize.py

```diff
@@ -0,0 +1,57 @@
+def precalc():
+	A.sort()
+	p_sum.append(A[0])
+	p_square_sum.append(A[0] * A[0])
+	for i in xrange(1, N):
+		p_sum.append(p_sum[i - 1] + A[i])
+		p_square_sum.append(p_square_sum[i - 1] + (A[i] * A[i]))
+
+def min_error(lo, hi):
+	sum_ = p_sum[hi]
```

Comment:
to keep consistency with JMB, if-else clause might work better.
e.g. )  sum = p_sum[hi] - (0 if lo == 0 else p_sum[lo - 1])